pygeoapi/provider/xarray_edr.py

```diff
@@ -252,7 +261,7 @@ def _make_datetime(self, datetime_):
                 LOGGER.debug('Reversing slicing from high to low')
                 return slice(end, begin)
         else:
-            return datetime_
+            return pd.Timestamp(datetime_)
```

Comment:
Are we able to do this without pandas here?

Reply:
I'll investigate a bit further. The reason I did it this way is because [xarray's docs mention how its datetime indexing borrwos a lot from pandas](https://docs.xarray.dev/en/stable/user-guide/time-series.html#datetime-indexing) and also because pandas was already listed as a requirement in `requirements-provider.txt`. Maybe I can just use some analogous numpy datatype to get the same result, I'll check it out

Reply:
tests pass by replacing `pd.Timestamp` with `np.datetime64`, so I switched to numpy